linter.py

```diff
@@ -25,13 +26,13 @@ class Flow(Linter):
     version_requirement = '>= 0.1.0'
     regex = r'''(?xi)
         # Warning location and optional title for the message
-        ^.+/(?P<file_name>[^/]+\.(js|html)):(?P<line>\d+):(?P<col>\d+),\d+:\s*(?P<message_title>.+)$\r?\n
+        ^.+\/(?P<file_name_1>.+):(?P<col_1>(\d+:\d+,(\d+:)?\d+)):\s(?P<message_title>.+)?\r?\n
 
-        # Main lint message
-        ^(?P<message>.+)$
+        # (Optional) main message
+        (^(?P<message>.+))?
 
-        # Optional message, only extract the text, leave the path
-        (\r?\n\s\s/.+:\s(?P<message_footer>.+))?
+        # (Optional) message footer
+        (\r?\n\s\s.+\/(?P<file_name_2>.+):(?P<col_2>(\d+:\d+,(\d+:)?\d+)):\s(?P<message_footer>.+))?
```

Comment:
It could be nice to update the second message to include the origin file name and line number but the regex should resemble the original "Warning location and optional title for the message" regex.